code/website/index.html

```diff
@@ -3,11 +3,19 @@ <h2>Drasil - Generate All the Things!</h2>
 <hr>
 <br>
 <p>
+  Drasil is a framework for generating all of the software artifacts from a stable knowledge base,
+  focusing currently on scientific software. The main goals are to reduce knowledge duplication and
+  improve traceability. The atifacts are generated from a common knowledge-base using recipes
+  written in a Domain-Specific Language (DSL). These recipes allow us to specify which pieces of
+  knowledge should be used in which artifacts, how to transform them, and more.
+  <br><br>
   This webpage is designed to contain the most up to date
-  <a href="#caseStudies">case study artifacts</a>
+  <a href="#caseStudies">case study artifacts</a>,
+  $if(docsUrl)$ <a href="#haskDoc">Haddock documentation</a>, $endif$
   and <a href="#packages">package dependency graphs</a> from the Drasil repository.
   The case study artifacts include the Software Requirements Specification (SRS) for the case study,
   which specifies what the program sets out to achieve.
+  The Haskell Documentation section contains the current documentation for the Drasil framework.
```

Comment:
Haskell -> Haddock.  Lack of traceability / coherence caused this inconsistency to appear!

Reply:
Fixed in 4528e776df7bc7133e8046640ebca8280681222e